src/main/java/org/jabref/gui/entryeditor/fileannotationtab/FileAnnotationTabViewModel.java

```diff
@@ -35,6 +35,7 @@
     private final ListProperty<FileAnnotationViewModel> annotations = new SimpleListProperty<>(FXCollections.observableArrayList());
     private final ListProperty<Path> files = new SimpleListProperty<>(FXCollections.observableArrayList());
     private final ObjectProperty<FileAnnotationViewModel> currentAnnotation = new SimpleObjectProperty<>();
+    private final ObjectProperty<Boolean> annotaionEmpty = new SimpleObjectProperty<Boolean>(null);
```

Comment:
There's a spelling mistake in the variable's name here. It should be `annotationEmpty`.